master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala

```diff
@@ -45,11 +46,13 @@ class MasterArguments(args: Array[String], conf: CelebornConf) {
     _host = _host.orElse(Some(conf.haMasterNodeHost(localNode.nodeId)))
     _port = _port.orElse(Some(conf.haMasterNodePort(localNode.nodeId)))
     _internalPort = _internalPort.orElse(Some(conf.haMasterNodeInternalPort(localNode.nodeId)))
+    _securedPort = _securedPort.orElse(Some(conf.haMasterNodeSecuredPort(localNode.nodeId)))
```

Comment:
Same as https://github.com/apache/incubator-celeborn/pull/2283

Reply:
I fixed this. Thanks for letting me know.